Remotion/Build/Customizations/Projects.props

```diff
@@ -2,16 +2,51 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <PropertyGroup>
-    <TestExecutionRuntime>net-4.6.1</TestExecutionRuntime>
+    <TestSetupBuildFileForDatabase>$(SolutionDirectory)Build\TestingSetupForDatabase.targets</TestSetupBuildFileForDatabase>
+    <TestSetupBuildFileForWebTest>$(SolutionDirectory)Build\TestingSetupForWebTest.targets</TestSetupBuildFileForWebTest>
   </PropertyGroup>
 
   <PropertyGroup>
-    <TestingSetupBuildFileForDatabase>$(SolutionDirectory)Build\TestingSetupForDatabase.targets</TestingSetupBuildFileForDatabase>
-    <TestingSetupBuildFileForWebTest>$(SolutionDirectory)Build\TestingSetupForWebTest.targets</TestingSetupBuildFileForWebTest>
+    <WebtestingTestConfiguration Condition="'$(WebtestingTestConfiguration)' == ''">
+      Chrome           + NET472 + Debug   + x86 + NoDB + EnforcedLocalMachine;<!-- Upgrade to NET48 once it is supported -->
+      Firefox          + NET472 + Release + x64 + NoDB + EnforcedLocalMachine;
+      InternetExplorer + NET472 + Release + x64 + NoDB + EnforcedLocalMachine;<!-- Upgrade to NET48 once it is supported -->
+      Edge             + NET472 + Release + x64 + NoDB + EnforcedLocalMachine;<!-- Upgrade to NET48 once it is supported -->

```

Comment:
Since these webtests are run in NET472 we currently cannot run them on agent 2 as long as it doesn't have .NET Framework 4.7.2 like the other agents